src/NetPanzer/Views/Game/VehicleSelectionView.cpp

```diff
@@ -590,7 +591,7 @@ void VehicleSelectionView::drawMiniProductionStatus(Surface &dest)
             {
                 strncpy(strBuf, obj->occupying_player->getName().c_str() , 17);
                 strBuf[17] = 0; // fix runners
-                sprintf(outpostUserNameBuf, "Owner: %s...", strBuf);
+                sn_check(std::snprintf ( outpostUserNameBuf, sizeof outpostUserNameBuf, "Owner: %s...", strBuf ), sizeof outpostUserNameBuf );
```

Comment:
The reason I don't call snprintf() from the macro is so that sn_check() can be used in other cases where one would want to just compare two values without using snprintf, which returns a value.